src/components/AnchorTag/AnchorTag.jsx

```diff
@@ -14,3 +15,9 @@ export default function AnchorTag({ anchorText, url, anchorKey = '' }) {
     </a>
   );
 }
+
+AnchorTag.propTypes = {
+  anchorText: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+  anchorKey: PropTypes.string
+}
```

Comment:
Recurda agregar una linea al final de cada archivo para seguir el estandar end of line: 'lf'